diff --git glibc-clean/Makeconfig glibc-workdir/Makeconfig
index 2fda4af..5b29b04 100644
--- glibc-clean/Makeconfig
+++ glibc-workdir/Makeconfig
@@ -262,7 +262,7 @@ inst_sysconfdir = $(install_root)$(sysconfdir)
 
 # Directory for the database files and Makefile for nss_db.
 ifndef vardbdir
-vardbdir = $(localstatedir)/db
+vardbdir = $(localstatedir)/lib/nss_db
 endif
 inst_vardbdir = $(install_root)$(vardbdir)
 
diff --git glibc-clean/nscd/nscd.h glibc-workdir/nscd/nscd.h
index 04f82fc..fd72114 100644
--- glibc-clean/nscd/nscd.h
+++ glibc-workdir/nscd/nscd.h
@@ -160,11 +160,11 @@ struct database_dyn
 
 
 /* Paths of the file for the persistent storage.  */
-#define _PATH_NSCD_PASSWD_DB	"/var/db/nscd/passwd"
-#define _PATH_NSCD_GROUP_DB	"/var/db/nscd/group"
-#define _PATH_NSCD_HOSTS_DB	"/var/db/nscd/hosts"
-#define _PATH_NSCD_SERVICES_DB	"/var/db/nscd/services"
-#define _PATH_NSCD_NETGROUP_DB	"/var/db/nscd/netgroup"
+#define _PATH_NSCD_PASSWD_DB	"/var/cache/nscd/passwd"
+#define _PATH_NSCD_GROUP_DB	"/var/cache/nscd/group"
+#define _PATH_NSCD_HOSTS_DB	"/var/cache/nscd/hosts"
+#define _PATH_NSCD_SERVICES_DB	"/var/cache/nscd/services"
+#define _PATH_NSCD_NETGROUP_DB	"/var/cache/nscd/netgroup"
 
 /* Path used when not using persistent storage.  */
 #define _PATH_NSCD_XYZ_DB_TMP	"/var/run/nscd/dbXXXXXX"
diff --git glibc-clean/nss/db-Makefile glibc-workdir/nss/db-Makefile
index 0dcf764..861e62e 100644
--- glibc-clean/nss/db-Makefile
+++ glibc-workdir/nss/db-Makefile
@@ -21,7 +21,7 @@ DATABASES = $(wildcard /etc/passwd /etc/group /etc/ethers /etc/protocols \
 		       /etc/rpc /etc/services /etc/shadow /etc/gshadow \
 		       /etc/netgroup)
 
-VAR_DB = /var/db
+VAR_DB = /var/lib/nss_db
 
 AWK = awk
 MAKEDB = makedb --quiet
diff --git glibc-clean/sysdeps/generic/paths.h glibc-workdir/sysdeps/generic/paths.h
index 893b4c2..ae9eb27 100644
--- glibc-clean/sysdeps/generic/paths.h
+++ glibc-workdir/sysdeps/generic/paths.h
@@ -68,7 +68,7 @@
 /* Provide trailing slash, since mostly used for building pathnames. */
 #define	_PATH_DEV	"/dev/"
 #define	_PATH_TMP	"/tmp/"
-#define	_PATH_VARDB	"/var/db/"
+#define	_PATH_VARDB	"/var/lib/nss_db/"
 #define	_PATH_VARRUN	"/var/run/"
 #define	_PATH_VARTMP	"/var/tmp/"
 
diff --git glibc-clean/sysdeps/unix/sysv/linux/paths.h glibc-workdir/sysdeps/unix/sysv/linux/paths.h
index 1342ab3..295505a 100644
--- glibc-clean/sysdeps/unix/sysv/linux/paths.h
+++ glibc-workdir/sysdeps/unix/sysv/linux/paths.h
@@ -68,7 +68,7 @@
 /* Provide trailing slash, since mostly used for building pathnames. */
 #define	_PATH_DEV	"/dev/"
 #define	_PATH_TMP	"/tmp/"
-#define	_PATH_VARDB	"/var/db/"
+#define	_PATH_VARDB	"/var/lib/nss_db/"
 #define	_PATH_VARRUN	"/var/run/"
 #define	_PATH_VARTMP	"/var/tmp/"
 
